PORTING.md

```diff
@@ -0,0 +1,56 @@
+Porting applications to use zlib-ng
+-----------------------------------
+
+Zlib-ng can be used/compiled in two different modes, that require some
+consideration by the application developer.
+
+zlib-compat mode
+----------------
+Zlib-ng can be compiled in zlib-compat mode, suitable for zlib-replacement
+in a single application or system-wide.
+
+Please note that zlib-ng in zlib-compat mode is API-compatible but not
+ABI-compatible, meaning that you cannot simply replace the zlib library/dll
+files and expect the application to work. The application will need to be
+recompiled against the zlib-ng headers and libs to ensure full compatability.
+
+Zlib-ng can be distinguished from other zlib implementations at compile-time
+by checking the existence of `#ifdef ZLIBNG_VERSION` in the zlib.h header
```

Comment:
Might read better as "checking for the existence of the `ZLIBNG_VERSION` predefined macro in the zlib.h header." (period)